Rephrase partitions warning, and add to FAQs (#264)
harvester/docs: docs/faq.md

New password for default administrator (user-xxxxx):
<new_password>
```

> ### I added an additional disk, but why can the disk not get detected?

Reviewer: ```suggestion
### I added an additional disk with partitions. Why is it not getting detected?
```

I'd suggest making the question specific (mention partitions) to match the answer.
